cli/command/stack/kubernetes/stackclient.go

```diff
@@ -100,6 +103,18 @@ func verify(services corev1.ServiceInterface, stackName string, service string)
 	return nil
 }
 
+func (s *stackV1Beta1) LoadStack(name string, cfg composetypes.Config) (stack, error) {
```

Comment:
nit: not sure if `LoadStack` is the correct name for those too :angel:

Reply:
PTAL